### DISCUSSION TITLE
recursive path dependencies

### DISCUSSION THREAD

**Discussion by kholisrag** [gruntwork-io/knowledge-base#548] [Help] [answered]
does current terragrunt support recursive path dependencies ?

something like

from :

```hcl
dependencies {
  paths = [
    format("%s/../../vault-server/asg/config", get_terragrunt_dir()),
    format("%s/../../vault-server/asg/asg", get_terragrunt_dir()),
    format("%s/../../vault-server/nlb/public/nlb", get_terragrunt_dir())
  ]
}
```

to : 

```hcl
dependencies {
  paths = [
    format("%s/../../vault-server/*", get_terragrunt_dir()),
  ]
}
```

If not, this will be helpful feature tho...
Thanks!

---

<ins datetime="2022-08-30T17:52:17Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109189">Tracked in ticket #109189</a></p>
</ins>

**Comment by zackproser** [comment, marked as answer]
Hi @kholisrag - thanks for your question! 

It does not appear that Terragrunt currently supports the `*` wildcard operator in the way you've suggested here, but I could certainly see the appeal of that capability. I suppose it would trade clarity around the exact paths being searched for concision. 

I'd recommend opening an issue against the [Terragrunt repo](https://github.com/gruntwork-io/terragrunt/issues/new) so that our maintainers can consider your feature for inclusion.
